src/main/java/com/google/devtools/build/lib/exec/ExecutionOptions.java

```diff
@@ -351,6 +351,22 @@ public boolean shouldMaterializeParamFiles() {
   )
   public boolean localMemoryEstimate;
 
+  @Option(
+      name = "local_extra_resources",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      allowMultiple = true,
+      help =
+          "Explicitly set the number of extra recourses available to Bazel. "
+            + "Takes in a string-float pair. Can be used multiple times to specify multiple "
+            + "types of extra resources. Bazel will limit concurrently running test actions "
+            + "based on the available extra resources and the extra resources required "
+            + "by the test actions. "
+            + "Note: This is a no-op if --local_resources is set.",
```

Comment:
I wonder whether Bazel should fail when both are specified.

Reply:
Doesn't matter, `--local_resources` has been removed since the revision this is based on.